eZ/Publish/Core/Repository/Values/ContentType/FieldDefinition.php

```diff
@@ -116,7 +116,7 @@ public function getDescriptions()
      */
     public function getDescription($languageCode)
     {
-        if (array_key_exists($languageCode, $this->descriptions)) {
+        if (is_array($this->descriptions) && array_key_exists($languageCode, $this->descriptions)) {
```

Comment:
shouldn't the default value of descriptions be an array instead/as well since it's documented as `string[]` ?